src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -53,6 +53,7 @@ void LocalResourceManager::AddLocalResourceInstances(
 void LocalResourceManager::DeleteLocalResource(scheduling::ResourceID resource_id) {
   local_resources_.available.Remove(resource_id);
   local_resources_.total.Remove(resource_id);
+  resources_last_idle_time_.erase(resource_id);
```

Comment:
Let's do a sanity check to make sure we update resources_last_idle_time whenever OnResourceOrStateChanged is called.

Even better if we can update resources_last_idle_time inside OnResourceOrStateChanged so it's guaranteed that we don't miss anything.

Reply:
Ah, i see. i see this being a new function from your drain PR - we would want to pick this fix PR in 2.6.3, which might have some dependency with your drain PRs then?

Maybe we could address it and refactor later in master.

Reply:
Yea, don't have to do it now.